db/schema.rb

```diff
@@ -715,14 +715,14 @@
   add_index "teams_users", ["user_id"], name: "fk_teams_users", using: :btree
 
   create_table "track_notifications", force: :cascade do |t|
+    t.integer  "notification_id", limit: 4
```

Comment:
I don't think these changes to schema.rb are relevant to your project. Please revert them.

Reply:
Hi Akshay, I do not remember adding this line, I'm not very sure why it is showing up. Could you please delete this on our behalf. I do not want to mess up anything.